### ISSUE TITLE
testing

### ISSUE THREAD

**Issue by amrbashir** [tauri-apps/tauri-github-bot#85]

**Comment by amrbashir** [comment]
asdfasd

**Comment by lorenzolewis** [comment]
You doing okay there, bud?

**Comment by amrbashir** [comment]
testing the github bot using deno

**Comment by lorenzolewis** [comment]
asdfasdfewafef

**Comment by amrbashir** [comment]
aaaaaaaaaaaaaa

**Comment by lorenzolewis** [comment]
bbbbbbbbbb

**Comment by lorenzolewis** [comment]
Btw, if you're doing stuff with tags, is there a way to do it with a tag ID instead of the string/name? We're going to go through and rename (not replace) a bunch of tags across the org next week most likely to help with PMing stuff

**Comment by lorenzolewis** [comment]
Tags, labels, same thing

**Comment by lorenzolewis** [comment]
yeah, they don't give an ID through the API for them? (I'm assuming the bot uses the GitHub API?)

**Comment by amrbashir** [comment]
Yeah I am using GitHub API.

While they have Id for each label which I can retrieve, they are not useful because every operation on labels require the name of the label not the id.

**Comment by lorenzolewis** [comment]
Lack of autocomplete kills me at times
